parser.rb

```diff
@@ -54,22 +55,23 @@ def parse_players
 
   
   def parse_kills
-    re_pattern_kill = /(?:Kill:.*:)(.*)(?:killed)/
+    re_pattern_kill = /(?:Kill:.*:)(.*)(?:killed)(.*)(?:by)/
```

Comment:
Se considerar os espaços não vai precisar usar `.strip` lá na frente.
```suggestion
    re_pattern_kill = /(?:Kill:.*:) (.*) (?:killed) (.*) (?:by)/
```